src/javascript/app/base/footer.js

```diff
@@ -0,0 +1,22 @@
+const getElementById         = require('../../_common/common_functions').getElementById;
+
+const Footer = (() => {
+    const displayNotification = (message) => {
+        const status_notification = getElementById('status-notification');
+        const status_message_text = getElementById('status-notification-text');
+        const close_icon = getElementById('status-notification-close');
+
+        $(status_notification).css('display', 'flex');
+        status_message_text.html(message);
+
+        $(close_icon).click(() => {
```

Comment:
should we also handle off click to avoid multiple event listeners on the same element?